src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -732,6 +745,23 @@ private void ProcessPing(string targetNameOrAddress)
             return true;
         }
 
+        private IPHostEntry GetHostEntryWithCancel(string targetNameOrAddress)
+        {
+            _dnsLookupCancel = new CancellationTokenSource();
```

Comment:
Is the method called in all code paths? In the case please move the initializer in the field definition.

Reply:
I'm... not sure if it is, actually. But thinking about it, I'd be surprised if it was not. I'll move it. 🙂